package.json

```diff
@@ -6,14 +6,15 @@
     "packages/*"
   ],
   "scripts": {
-    "prebuild": "rimraf docs && lerna exec --parallel -- rimraf lib",
-    "build": "yarn build:sources && lerna exec --parallel -- typedoc --options ../../typedoc.js --out ../../docs/api/\\$LERNA_PACKAGE_NAME --readme README.md --tsconfig tsconfig.typedoc.json .",
+    "prebuild": "rimraf docs",
```

Comment:
Deleting `lib` before building is still valid, even without project references, isn't it?

Reply:
we delete it now before build:sources

Reply:
before we didn't do that because we broke the caching. that doesn't exist anymore, does it?

Reply:
ah, I missed that